packages/Card/src/components/Avatar/Avatar.js

```diff
@@ -0,0 +1,51 @@
+import React from "react";
+import PropTypes from "prop-types";
+import { ShirtSizes } from "@paprika/helpers/lib/customPropTypes";
+import * as sc from "./Avatar.styles";
+
+const propTypes = {
+  /** Avatar content. It can be initial as a string or icon */
+  children: PropTypes.node,
+  /** Background color of the Avatar */
+  color: PropTypes.oneOf(["black", "blue", "green", "orange", "pink", "red"]),
```

Comment:
should not be a defined colour for the background, the colour can be any string, we will not validate that the string passes down its a valid colour that its responsibility of the consumer.